src/sw.js

```diff
@@ -0,0 +1,210 @@
+/**
```

Comment:
I might be missing something, but is the service worker caching the flavor's static assets? I think everything below the `/static/` folder should be fetched and cached.

Reply:
Thanks for looking into this! I just pushed up a commit that allows us to cache the `/static` routes: https://github.com/mapseed/platform/pull/1171/commits/709cc29ac532aebb076e67b2516e3bc0def1c05f

It's still not fixing the issue though. But I'm interested to see how you are testing it. Perhaps we can chat later this week.

Reply:
Sounds good-- yeah, it might be easiest to compare notes IRL later this week.